mypyc/lib-rt/bytes_ops.c

```diff
@@ -23,6 +23,32 @@ PyObject *CPyBytes_Concat(PyObject *a, PyObject *b) {
     }
 }
 
+#define CLAMP(a, b, c) (a < b ? b : (a >= c ? c : a))
```

Comment:
Style nit: use an inline function, since it will likely perform the same but is less error-prone.